app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, file, directory = command.split()
```

Comment:
you should check length of command.split before unpacking in order to avoid unexpected error